documentation/userref/config/job_params_cfg.rst

```diff
@@ -40,6 +40,11 @@ The parameters in this file are defined as follows:
    Integer. Determines the size of the subsection used for RMS measurement:
    the central ``1/f`` of the image will be used (where f=rms_est_fraction).
 
+``bandwidth``
```

Comment:
--> 'bandwidth_max'
I would maybe tweak the docstring a bit too:
"Limit the maximum bandwidth used when determining if two images belong to the same 'band' grouping."